data/reusables/dependabot/supported-package-managers.md

```diff
@@ -116,9 +117,11 @@ For {% data variables.product.prodname_dependabot_security_updates %}, Gradle su
 
 {% ifversion dependabot-PEP621-support %}
 
-#### pip and pip-compile
+#### pip-compile and uv
```

Comment:
why does this remove `pip`?

Reply:
i would just add `uv` to the end of it.  it is the "details" anchor used from the table above that includes details about how Dependabot supports `pip` overall.

https://docs.github.com/en/code-security/dependabot/ecosystems-supported-by-dependabot/supported-ecosystems-and-repositories#pip-and-pip-compile